### ISSUE TITLE
Warning: 1.3 deprecated function in Introduction.Ipynb

### ISSUE THREAD

**Issue by stonebig** [ioam/holoviews#213]
````
WARNING:root:Deprecation Warning: Layout.collate will be deprecated after version 1.3.0.
Use HoloMap.collate instead (see HoloViews homepage for example usage)
````
![holo_introduction_deprecation](https://cloud.githubusercontent.com/assets/4312421/8634114/ae43427c-27eb-11e5-92c1-696d79d6e73c.GIF)

**Comment by philippjfr** [comment]
Thanks, will fix that now.

**Comment by stonebig** [comment]
will all that still work with coming Matplotlib 1.5 at the end of the month ?
(they did add a lot of interactivity, maybe there could be some issues ?)

**Comment by philippjfr** [comment]
I'll test everything with the current matplotlib dev version in the coming days but I see no reason why it shouldn't. The added features for interactivity should actually be a good thing for us as it will allow us to significantly speed up certain things in HoloViews.

**Comment by stonebig** [comment]
Certainly. nice talk at scipy.

**Comment by philippjfr** [comment]
Thanks! Just pushed the fix to the Tutorial (7b043b87). Will close the issue now.
